src/pmpo_defines.h

```diff
@@ -23,7 +25,7 @@ using kkViewHostU = Kokkos::View<
           Kokkos::MemoryTraits<Kokkos::Unmanaged>>;
 
 typedef kkViewHostU<double*> kkDblViewHostU;//TODO:put it to mesh.hpp             
-typedef kkViewHostU<double*[vec2d_nEntries]> kkVec2dViewHostU;//TODO:put it to mesh.hpp
+typedef kkViewHostU<polyMPO::vec2d_t*> kkVec2dViewHostU;//TODO:put it to mesh.hpp
```

Comment:
We need to clean these up and will require some discussion. For example, do we need kkVec3dViewHostU (like kkVec2dViewHostU). Please open an issue on this for now.